src/index.ts

```diff
@@ -22,7 +22,7 @@ interface GraphData {
   currentNoteID: string;
   nodeFontSize: number;
   nodeDistanceRatio: number;
-  isIncludeBacklinks: boolean;
```

Comment:
I don't think you intended to remove this feature. also can we call is `shouldIncludeBacklinks`